python/paddle/fluid/layers/nn.py

```diff
@@ -4799,9 +4802,7 @@ def image_resize_short(input, out_short_len, resample='BILINEAR'):
 
 def gather(input, index):
     """
-    **Gather Layer**
-
-    Output is obtained by gathering entries of the outer-most dimension 
+    Output is obtained by gathering entries of the outer-most dimension
```

Comment:
Seem that there is some diff in the annotation of `nn.py`, do you modify this file? If not, you could remain it.